docs/FAQs/Masternode-FAQ.md

```diff
@@ -2,7 +2,7 @@
 
 ##### Q: How many coins for a Masternode, and how do I set one up?
 
-A: 5000 BWK, Check out the pinned messages on #masternodes for a setup guide from our developer Stu.
+A: You will need a total of 5000 BWK. Follow this Guide to set one up: [Bulwark MN Install](https://github.com/bulwark-crypto/Bulwark-MN-Install)

```

Comment:
I'd link "guide" here instead of adding the link at the end - `follow this [guide](...)`